src/components/Sidebar/Sidebar.tsx

```diff
@@ -12,7 +12,7 @@ import SocialLinks from '../SocialLinks/SocialLinks';
 const SidebarContainer = styled.aside`
   display: flex;
   flex-direction: column;
-  margin-top: 20px;
+  margin-top: 10px;
```

Comment:
Can this go back to the way it was now?  Or is this margin change impacting the banner in some way?

```suggestion
  margin-top: 20px;
```